Offseason2022/src/main/java/frc/robot/commands/VisionOn.java

```diff
@@ -27,6 +27,17 @@ public VisionOn(Vision vision, boolean lightOn)
   public void initialize( )
   {
     m_vision.setLEDMode((m_lightOn) ? VIConsts.LED_ON : VIConsts.LED_OFF);

```

Comment:
If I walk through the logic on this, line 29 SETS the LL LED to on/off based on what the caller passes into this method. Then on line 31, it reads that same LL LED setting back from the vision subsystem (which will be what the caller wanted to happen). It uses that reading to invert it again. This probably happened because some of the code was copied from VisionOnToggle. It doesn't quite work, and could be better.

I propose this command take a parameter that is LED_ON, LED_OFF, and LED_TOGGLE. This will handle all of the cases we want. You can make a switch statement to decode the input parameter OR just use a couple of if statements. Then use that info to either turn LED_ON/PIP_SECONDARY, LED_OFF/PIP_MAIN, OR read the existing value and flip both settings LED_xxx/PIP_xxx.